javascripts/book-download.js

```diff
@@ -0,0 +1,45 @@
+/*
+  This script is responsible for book downloads.
+
+  The script is only active if the page was visited via a redirect from the download manager. It uses the query-string
+  provided by the download-manager to know which file to download.
+ */
+app.bookDownload = {
+  urlParam: function (name) {
+    var results = new RegExp('[\\?&]' + name + '=([^&#]*)').exec(window.location.href);
+    if (!!results) {
+      return results[1] || 0;
+    } else {
+      return null;
+    }
+  },
+  download: function () {
+    // set when signed to blank when we do not require a login
+    var tcDownloadURL = $.encoder.canonicalize(app.bookDownload.urlParam('tcDownloadURL'));
+    var tcDownloadFileName = $.encoder.canonicalize(app.termsAndConditions.urlParam('tcDownloadFileName'));
+
+    if (tcDownloadURL &&
+        tcDownloadURL.startsWith('https://access.cdn.redhat.com/')) {
+      tcDownloadURL = $.encoder.canonicalize(window.location.href.substr(window.location.href.indexOf("tcDownloadURL=") + 14));
+
+      $("a#tcDownloadLink").attr("href", tcDownloadURL);
+
+      $.fileDownload(tcDownloadURL);
+
+      // Inform GTM that we have requested a product download
+      window.dataLayer = window.dataLayer || [];
+      window.dataLayer.push({ 'product_download_file_name' : tcDownloadFileName });
+      window.dataLayer.push({'event': 'Product Download Requested'});
```

Comment:
Needs to be:

        window.dataLayer.push({'event': 'Book Download Requested'});